src/base/DelegateInputLabelMixin.js

```diff
@@ -31,18 +32,60 @@ export default function DelegateInputLabelMixin(Base) {
       this[setState]({ ariaLabel });
     }
 
+    // Forward ARIA labelledby as an aria-label to the input element.
+    get ariaLabelledby() {
+      return this[state].ariaLabelledby;
+    }
+    set ariaLabelledby(ariaLabelledby) {
+      if (this.getRootNode() !== null) {
+        // @ts-ignore
+        let labelNode = this.getRootNode().querySelector(
```

Comment:
A more direct way to do this search is `getElementById`.

Reply:
Prefer `const` over `let` here, since the `labelNode` isn't changing later.

Reply:
Also, `aria-labelledby` can take a space-delimited set of IDs, so this will need to be a loop.